### PR TITLE
Teardown Project Review Request

### PR DESCRIPTION
**Pull request by JelilFaisalAbudu**
# Teardown Project Review Request

- All the features were implemented;
- Did a grayscale and not a clone of the project to study UX;
- HTML Validator checked 
-CSS linter checked.
- Awaiting for code review from TSE:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Teardown Project</title>
+</head>
+
+<body>
+    <header class="header">
```
Comment (uryelah) [inline index.html]: - [x] Use the `<nav>` tag for the nav bar. It's more semantic.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+
+        </div>
+    </header>
+
+    <article class="container-top">
+        <div class="more-articles">
+            <p>Don't miss these latest<br>articles.</p>
+        </div>
+        <div class="change-colors"></div>
+        <div class="top-box">
```
Comment (uryelah) [inline index.html]: - [x] `<div>`s of the .top-box class and other divs that represent articles should use the `<article>` tag. Make sure you check all the `div`s that have self-contained element representing an article and use the semantic tag for them.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+            </div>
+
+
+        </div>
+        <div class="see-all-latest-articles-title design-see-all">
+            <span>see all the articles</span> 
+        </div>
+    </article>
+
+    <footer class="footer">
```
Comment (uryelah) [inline index.html:419]: - [x] The footer should contain centered **links**. Remember to add the `<a>` tags.

**Review by uryelah — CHANGES_REQUESTED** [review]
Hello **Faisal Abudu** & **Felipe Dacal Fragoso**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, good job with this project.🎇**
You guys did a great job paying attention to the page elements and adding even the transformations.

There are still some minor issues to work through so hang in there 💪 

---------    

### Required changes 🛠 
---------
**Check the required changes in the comments below** 🔽

**Comment by uryelah** [comment]
**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by uryelah** [comment]
Great job with the required changes.
Your project is good enough to pass.

## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

-------

### Optional Improvements

- [x] [OPTIONAL] For the `<article>` tags that contain multiple articles within themselves change the tag to `<section>` for a more semantic HTML. [ Check this MDN article with examples of a good semantic HTML structure.](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines)
-------

**Keep being an awesome dev, good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: index.html [hunk 4]
```diff
@@ -23,7 +23,7 @@ <h2>
             <h3>
                 Lorem ipsum
             </h3>
-        </div>
+        </artic>
```
Comment (fdfragoso) [inline index.html:26]: article and not artic

**Pull request merged**